cmd/util/cmd/epochs/cmd/recover.go

```diff
@@ -20,22 +21,20 @@ import (
 // identities, generates the cluster QC's and retrieves the DKG key vector of the last successful epoch.
 var (
 	generateRecoverEpochTxArgsCmd = &cobra.Command{
-		Use:   "generate-efm-recovery-data",
+		Use:   "efm-recover-tx-args",
 		Short: "Generates recover epoch transaction arguments",
 		Long:  "Generates transaction arguments for the epoch recovery transaction.",
```

Comment:
```suggestion
		Long:  `
Generates transaction arguments for the epoch recovery transaction.
The epoch recovery transaction is used to recover from any failure in the epoch transition process without requiring a spork.
This recovery process has some constraints:
- The RecoveryEpoch must have exactly the same consensus committee as participated in the most recent successful DKG.
- The RecoveryEpoch must contain enough "internal" collection nodes so that all clusters contain a supermajority of "internal" collection nodes (same constraint as sporks)
`,
```